### PR TITLE
Release nodejs-pubsub v0.20.0

### PR DESCRIPTION
**Pull request by jkwlui**
This pull request was generated using releasetool.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+### Internal / Testing Changes
+
+- Update the CI config (#220)
+- chore: make the CircleCI config consistent
+- chore: use arrow functions (#215)
+- chore: convert to es classes (#211)
+- chore: do not use npm ci (#209)
+- chore: use let and const (#204)
+- chore: ignore package-lock.json (#207)
+- chore: use split common modules (#200) - chore: update renovate config (#199)
```
Comment (JustinBeckwith) [inline CHANGELOG.md]: missing a newline :)

**Comment by kinwa91** [comment]
my bad - it turns out there's more things since 0.19.0 was last released.

I will make a patch release 0.19.1 with just the doc change.

**Review by JustinBeckwith — APPROVED** [review]
LGTM w/nit

**Comment by kinwa91** [comment]
Release is at https://github.com/googleapis/nodejs-pubsub/releases/tag/v0.20.0

**Pull request merged**